tests/init_test.py

```diff
@@ -17,7 +17,7 @@ def test_metadata():
     latest_version = list(releases.items())[-1]
 
     assert DiffPriv.__name__ == 'DiffPriv'
-    assert DiffPriv.__version__ == 'v1.0.1'
+    assert DiffPriv.__version__ == 'v1.0.3'
```

Comment:
Tests should be version-agnostic; delete this.

Reply:
Adding in branch `version-fix`